test/ngAnimate/animateCssSpec.js

```diff
@@ -51,6 +51,43 @@ describe("ngAnimate $animateCss", function() {
   describe('when active', function() {
     if (!browserSupportsCssAnimations()) return;
 
+    it("should silently quit the animation and not throw when an element is not attached to a parent during preparation",
```

Comment:
`when an element has no parent` removes the potential confusion of a double negative.